.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+
+
+---
+
+Before we can merge this PR, please make sure that all the following items have been checked off:
+
+- [ ] Linked to an issue with discussion and accepted design OR have an explanation in the PR that describes this work.
+- [ ] Added **one** line describing your change in [`CHANGELOG.md`](https://github.com/manta-network/blob/main/CHANGELOG.md) and added the appropriate `changelog` label to the PR.
```

Comment:
Shouldn't it be `https://github.com/manta-network/manta-rs/blob/main/CHANGELOG.md`?

Reply:
Good catch thanks!